packages/core/admin/admin/src/content-manager/pages/ListView/components/AutoCloneFailureModal.tsx

```diff
@@ -0,0 +1,147 @@
+import * as React from 'react';
+
+import {
+  Box,
+  Button,
+  Flex,
+  Icon,
+  ModalBody,
+  ModalFooter,
+  ModalHeader,
+  ModalLayout,
+  Typography,
+} from '@strapi/design-system';
+import { LinkButton } from '@strapi/design-system/v2';
+import { ChevronRight } from '@strapi/icons';
+import { useIntl } from 'react-intl';
+import { useLocation, NavLink } from 'react-router-dom';
+
+import { getTranslation } from '../../../utils/translations';
+
+import type { Entity } from '@strapi/types';
+
+type Reason = 'relation' | 'unique';
+type ProhibitedCloningField = [string[], Reason];
+
+interface AutoCloneFailureModalProps {
+  onClose: () => void;
+  entryId: Entity.ID | null;
+  prohibitedFields: ProhibitedCloningField[];
+  pluginQueryParams: string;
+}
+
+const AutoCloneFailureModal = ({
+  onClose,
+  entryId,
+  prohibitedFields,
+  pluginQueryParams,
+}: AutoCloneFailureModalProps) => {
+  const { formatMessage } = useIntl();
+  const { pathname } = useLocation();
+
+  if (!entryId) {
+    return null;
+  }
+
+  const editPath = `${pathname}/create/clone/${entryId}?${pluginQueryParams}`;
+
+  const getDefaultErrorMessage = (reason: Reason) => {
+    switch (reason) {
+      case 'relation':
+        return 'Duplicating the relation would remove it from the original entry.';
+      case 'unique':
+        return 'Identical values in a unique field are not allowed';
+      default:
+        return reason;
+    }
+  };
+
+  return (
+    <ModalLayout onClose={onClose} labelledBy="title">
+      <ModalHeader>
+        <Typography variant="omega" fontWeight="bold" as="h2" id="title">
+          {formatMessage({
+            id: getTranslation('containers.ListPage.autoCloneModal.header'),
+            defaultMessage: 'Duplicate',
+          })}
+        </Typography>
+      </ModalHeader>
+      <ModalBody>
+        <Typography variant="beta">
+          {formatMessage({
+            id: getTranslation('containers.ListPage.autoCloneModal.title'),
+            defaultMessage: "This entry can't be duplicated directly.",
+          })}
+        </Typography>
+        <Box marginTop={2}>
+          <Typography textColor="neutral600">
+            {formatMessage({
+              id: getTranslation('containers.ListPage.autoCloneModal.description'),
+              defaultMessage:
+                "A new entry will be created with the same content, but you'll have to change the following fields to save it.",
+            })}
+          </Typography>
+        </Box>
+        <Flex marginTop={6} gap={2} direction="column" alignItems="stretch">
+          {prohibitedFields.map(([fieldPath, reason]) => (
+            <Flex
+              direction="column"
+              gap={2}
+              alignItems="flex-start"
+              borderColor="neutral200"
+              hasRadius
+              padding={6}
+              key={fieldPath.join()}
+            >
+              <Flex direction="row" as="ul">
```

Comment:
I think it should be ordered list `ol` because the order of the items matters right? it's a path to the field... 🤔